src/node_crypto.cc

```diff
@@ -4320,7 +4360,7 @@ SignBase::Error Verify::VerifyFinal(const char* key_pem,
   if (!initialised_)
     return kSignNotInitialised;
 
-  ClearErrorOnReturn clear_error_on_return;
+  // ClearErrorOnReturn clear_error_on_return;
 
```

Comment:
so i comment out this line for now. Maybe i'm wrong, but it feels like this was in the wrong spot. This is not the binding function and I thought the openssl's error stack needs to be balanced upon entry and exit of the VerifyFinal bound function. So i moved it there. Please let me know if im wrong and I'll uncomment this and remove the line from the bound function, but I might need a little guidance on what I need to move around to not clear out the error stack when exiting this function. Maybe we just throw the error from this function instead of the bound function?